coderd/database/queries/provisionerdaemons.sql

```diff
@@ -69,6 +72,10 @@ LEFT JOIN
 			LIMIT 1
 		)
 	)
+JOIN
+	template_versions version ON version.id::text = pd.version
```

Comment:
There's no correlation between provisioner daemon and template versions. If you need this query to output an associated template/template version, the only option as I see it is to tie it to "current job" via `(current_job.input->>'template_version_id')::uuid` (it's a jsonb field).